Added Pod File

SunnyGurnani opened the pull request:
With this anyone can just add following into their pod file and everything else should be taken cared off.

I am not an expert in this but it worked.

pod 'react-native-cognito', :path => '../node_modules/react-native-cognito/ios'

SunnyGurnani committed:
Added Pod file

SunnyGurnani committed:
Moved to iOS directory

morcmarc commented:
Hey, thanks for the pull request! Good stuff, before I merge it in, could you bump the version to 1.3.0alpha, please? I want to make sure it is the same as in package.json.

SunnyGurnani committed:
changed version to alpha

SunnyGurnani committed:
Returning CognitoId

SunnyGurnani committed:
Passing back CognitoId

SunnyGurnani committed:
Changed to pass Object instead.

SunnyGurnani committed:
Removed Method Return type

SunnyGurnani committed:
Added Method to get CognitoId

SunnyGurnani committed:
Documentation for getCognitoId

SunnyGurnani commented:
Hey,
I have changed the version and also added getCognitoId method in IOS.

If someone can add the same method in Android as well it would be great.

SunnyGurnani committed:
Added getCognitoCredentials.

SunnyGurnani committed:
Added support to get CognitoId with unauthenticated user as well

SunnyGurnani committed:
Update package.json

SunnyGurnani committed:
Update react-native-cognito.podspec

SunnyGurnani committed:
Fixed Cognito Weirdness

SunnyGurnani committed:
added S3 upload

SunnyGurnani committed:
Reset and added upload to S3

SunnyGurnani committed:
Added import for S3

SunnyGurnani committed:
added support for content type

morcmarc commented:
Hi, I'm sorry but I can't merge this pull request. Features related to S3 should go into a separate plugin, I only want to focus on Cognito. Also did some research on using CocoaPods and it looks like the preferred way of distributing RN packages is NPM. See #4.

However I do appreciate your effort and have added you to the contributors ;)
